python/cugraph/linear_assignment/lap.py

```diff
@@ -69,9 +71,46 @@ def hungarian(G, workers):
     else:
         local_workers = workers
 
-    df = lap_wrapper.hungarian(G, local_workers)
+    df = lap_wrapper.sparse_hungarian(G, local_workers)
 
     if G.renumbered:
         df = G.unrenumber(df, 'vertex')
 
     return df
+
+
+def dense_hungarian(costs, num_rows, num_columns):
+    """
+    Execute the Hungarian algorithm against a dense bipartite
+    graph representation.
+
+    The Hungarian algorithm identifies the lowest cost matching of vertices
+    such that all workers that can be assigned work are assigned exactly
+    on job.
+
+    Parameters
+    ----------
+    costs : cudf.Series
```

Comment:
Do we expect users to store dense matrices in cudf Series? Other alternatives would be cupy I suppose.
Ideally, we would need to figure out the dense class/type for cugraph before having too many people using this API. I think we should at least add a disclaimer that this API is not stable yet.

Reply:
I think really that we need to follow the cuML model.  They have a python class that wraps ALL things that can look like an array (device or host side) and facilitates using those across to the C++ side.

My hope was to defer that complexity to when @rlratzel is ready for that but to still get out something that could be used.

I'd be happy to add to the documentation that this is an unstable API

Reply:
The cuML approach sounds good and I agree there's value in getting this out sooner so it can be used while a cuML-style abstract array object can be developed. I think the disclaimer is good here as well.